src/serac/physics/utilities/quadrature_data.hpp

```diff
@@ -46,6 +86,15 @@ inline MaybeOwningPointer<mfem::QuadratureFunction> initialQuadFunc(mfem::Quadra
   }
 }
 
+template <auto offsetV>
```

Comment:
I assume this is grabbing private data from the quadrature space class. Some comments reflecting this would be appreciated.